inclusive-design/wecount.inclusivedesign.ca
# feat: implement the new design for the Views page (resolves #459)

## Description

Implement the new design for the Views page.

## Steps to test

1. Go to Views page;
2. The layout should match [the figma design](https://www.figma.com/file/0lcLol3X5MmOXackT2YbHJ/WeCount-website?node-id=1622%3A6373);
3. Test with search and filter features. The results should be displayed in the layout of the new design.

**Expected behavior:** 

See above.
